patch/driver-support-optoe-twoaddr-a2h-access.patch

```diff
@@ -0,0 +1,57 @@
+From 31c2f29dd926d31b5d700f9f811aff3eaaefc5a9 Mon Sep 17 00:00:00 2001
+From: Jemston Fernando <jemston_fernando@dell.com>
+Date: Thu, 18 Nov 2021 00:22:50 -0800
+Subject: [PATCH] Enable 0x51(A2h) access even when DDM is unsupported, for
```

Comment:
Please add a commit message summary (cf. [How to Write a Git Commit Message](https://teatime.etherpad.molgen.mpg.de/p/0Mpm6ckyzxRipGyr2Kus)).

Reply:
Please add a space before *(A2h)*.

Reply:
Agreed.

Reply:
I updated the URL. Sorry about that.